File: tests/machine/x/cs/map_membership.cs
using System;
using System.Collections.Generic;

class Program {
    static void Main() {
        Dictionary<string, int> m = new Dictionary<string, int> { { "a", 1 }, { "b", 2 } };
        Console.WriteLine(m.ContainsKey("a"));
        Console.WriteLine(m.ContainsKey("c"));
    }
}

File: tests/human/x/rust/map_membership.rs
use std::collections::HashMap;
fn main() {
    let mut m: HashMap<&str, i32> = HashMap::new();
    m.insert("a", 1);
    m.insert("b", 2);
    println!("{}", m.contains_key("a"));
    println!("{}", m.contains_key("c"));
}
